Assets/Mirror/Tests/Editor/NetworkIdentityTests.cs

```diff
@@ -12,100 +12,105 @@ namespace Mirror.Tests
 {
     public class NetworkIdentityTests
     {
-        #region test components
+       #region test components
         class MyTestComponent : NetworkBehaviour
         {
             internal bool onStartServerInvoked;
 
-            public override void OnStartServer()
+            public void OnStartServer()
             {
                 onStartServerInvoked = true;
-                base.OnStartServer();
             }
         }
 
         class StartServerExceptionNetworkBehaviour : NetworkBehaviour
         {
             public int called;
-            public override void OnStartServer()
+            public void OnStartServer()
             {
                 ++called;
-                throw new Exception("some exception");
```

Comment:
The test was about checking what happens when the methods throw exceptions.

Reply:
From what I was seeing if it throws an exception the test will fail 100% of the time. I tried researching UnityEvents to see if I could find an answer as to why but found nothing. Today I was going to make a stupid simple test comparing the use of virtuals vs events and the same method called to see if throwing an exception is the only reason why the test would fail. If you have any other info in the mean time I am genuinely confused on this :)

Reply:
Resolved in recent commits. Please re-review.